bylaws/008-psr-evolution.md

```diff
@@ -0,0 +1,101 @@
+# PSR evolution
+
+## Scope and objectives
+
+A PSR is often comprised of text and code, generally interfaces. Those interfaces are pieces of code that are released and tagged at a specific moment in time. However, the PHP language doesn't stand still; it evolves over time.
+
+This means that those interfaces need to both provide a stable contract, as well as evolve to leverage new language features that could help better enforce the behaviors exposed in the PSR itself.
+
+At the same time, a PSR cannot be changed after its release (at which point only erratas are allowed), to protect a package that declared compatibility from becoming de-facto not compatible anymore.
+
+This document defines a process to be followed in updating PSR interfaces, in a way that is not breaking in regard to behavior for end users, and with an appropriate upgrade path for the consumers.
+
+## Definitions
+
+ * **Consumer** - libraries and projects that implement and/or consume the code released as part of the PSR in question.
```

Comment:
```suggestion
 * **Consumer** - a library or project that implements and/or consumes the code released as part of the PSR in question.
```

Reply:
One note on this part.  In PSRs, we tend to differentiate between an *Implementer* (who implements a PSR interface) and a *Consumer* (who composes it).  Strictly speaking both are "consumers" (in the economic sense) of a PSR, but I fear it may cause some confusion down the line.  Can we use the more standard terminology here, even if it makes it slightly longer?

Reply:
Done with aa3e198 :+1: